docs/best-practices/gitops-with-kruise.md

```diff
@@ -173,13 +92,143 @@ data:
 ```
 **OpenKruise internal CRD resources in addition to CloneSet, others such as: Advanced StatefulSet, SidecarSet, etc. can be similar to the above way to achieve Custom Resource Health.**
 
-### Argo-cd Sync (Deploy) The Application
-1. Complete the deployment of the guestbook service via the argo-cd command line tool (**argocd app sync guestbook**), as follows:
+### Tekton Pipeline + Argo-cd
+Argo-cd deployment application through Tekton Pipeline can better practice Devops thinking and connect with CI process.
```

Comment:
Argo-CD together with Tekton Pipeline is a popular DevOps practice and integrate well with CI process. Such practice requires ....